github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -837,6 +837,90 @@ periodics:
       - name: gcs
         secret:
           secretName: gcs
+- name: periodic-kubevirt-push-nightly-build-main-ARM64
+  cron: "2 1 * * *"
+  decorate: true
+  annotations:
+    testgrid-create-test-group: "false"
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+    preset-kubevirtci-quay-credential: "true"
+  extra_refs:
+    - org: kubevirt
+      repo: kubevirt
+      base_ref: main
+      work_dir: true
+  cluster: prow-workloads
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+    - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/gcs/service-account.json
+      - name: DOCKER_PREFIX
+        value: quay.io/kubevirt
+      - name: BUILD_ARCH
+        value: crossbuild-aarch64
+      command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/sh"
+        - "-c"
+        - |
+          set -e
+          cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
+          export DOCKER_TAG="$(date +%Y%m%d)_$(git show -s --format=%h)-arm64"
+          make
+          counter=3
+          retval=0
+          while [ $counter -gt 0 ]; do
+              set +e
+              make push
+              retval=$?
+              set -e
+              if [ $retval -eq 0 ]; then
+                  break
+              fi
+              set +e
+              counter=$(expr $counter - 1)
+              set -e
+              sleep 120
+          done
+          if [ $retval -ne 0 ]; then
+              echo "push failed!"
+              exit 1
+          fi
+          # build functest currently does not support crossbuild, skip it
+          git show -s --format=%H > _out/commit
+          build_date="$(date +%Y%m%d)"
```

Comment:
We probably have the same issue on the other build, but if we would run this job close to midnight, the `build_date` could be misleading. How about filling `build_date` at the beginning and using  it above in `DOCKER_TAG` too?